tools/codegen/model.py

```diff
@@ -123,7 +123,7 @@ def parse(value: str) -> 'DispatchKey':
         for k, v in DispatchKey.__members__.items():
             if k == value:
                 return v
-        raise AssertionError(f'unknown dispatch key {value}')
+        # raise AssertionError(f'unknown dispatch key {value}')
```

Comment:
A hacky way to introduce a loose mode is to have a custom assertion class which will check a global variable to suppress the assertion if that variable is set.

Reply:
Probably not enough to make the parser to be compatible with all historical native_functions.yaml.

Reply:
It looks like the further we go backward into the history. The more hacks are needed for the ToT parser. It doesn't seem to be sustainable to make the ToT parser to be compatible with all historic files.

I would suggest:
1. Temporarily use this new script combining hacks in the parser to generate all historical data.
2. Come out with a more future proof way of labeling the API availabilities.